HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1651,180 +1650,170 @@ def self.apply_hvac_installation_quality(hvac_sizing_values, weather, hvac)
-
-          cool_Load_SensCap_Design = (hvac_sizing_values.Cool_Capacity_Sens * sensibleCap_CurveValue /
-                                     (1.0 + (1.0 - coil_bf * bypassFactor_CurveValue) *
-                                     (80.0 - @cool_setpoint) / (@cool_setpoint - hvac.LeavingAirTemp)))
-          hvac_sizing_values.Cool_Airflow = calc_airflow_rate(cool_Load_SensCap_Design, (@cool_setpoint - hvac.LeavingAirTemp))
-        end
-      end
-
-      # Heating
-      if hvac.HeatingLoadFraction > 0
-        # Calculate curve point w/ and w/o defect ratios
-        design_db_temp = UnitConversions.convert(@heat_setpoint, 'f', 'k')
-        design_w_temp = UnitConversions.convert(hvac.GSHP_design_chw, 'f', 'k')
-        design_vfr_air = UnitConversions.convert(hvac_sizing_values.Heat_Airflow, 'cfm', 'm^3/s')
-        design_vfr_air_defect = UnitConversions.convert(hvac_sizing_values.Heat_Airflow, 'cfm', 'm^3/s') * (1 + hvac.AirflowDefectRatioHeating)
-        # calculate water flow based on current capacity.
-        loop_flow = [1.0, UnitConversions.convert([hvac_sizing_values.Heat_Capacity, hvac_sizing_values.Cool_Capacity].max, 'Btu/hr', 'ton')].max.floor * 3.0
-        loop_flow_m3s = UnitConversions.convert(loop_flow, 'gal/min', 'm^3/s')
+    f_ch = hvac.ChargeDefectRatio.round(3)
 
-        totalCap_CurveValue = calc_gshp_htg_curve_value(hvac, design_db_temp, design_w_temp, design_vfr_air, loop_flow_m3s)
-        totalCap_CurveValue_d = calc_gshp_htg_curve_value(hvac, design_db_temp, design_w_temp, design_vfr_air_defect, loop_flow_m3s)
+    # Cooling
+    if [HPXML::HVACTypeHeatPumpAirToAir,
+        HPXML::HVACTypeCentralAirConditioner,
+        HPXML::HVACTypeHeatPumpMiniSplit,
+        HPXML::HVACTypeMiniSplitAirConditioner,
+        HPXML::HVACTypeHeatPumpGroundToAir].include?(hvac.CoolType) && hvac.CoolingLoadFraction > 0
 
-        cap_htg_ratio = 1 / (totalCap_CurveValue_d / totalCap_CurveValue)
-        if cap_htg_ratio > 1
-          hvac_sizing_values.Heat_Capacity *= cap_htg_ratio
-          hvac_sizing_values.Heat_Airflow = calc_airflow_rate(hvac_sizing_values.Heat_Capacity, (hvac.SupplyAirTemp - @heat_setpoint))
-        end
-      end
-    else
-      f_ch = hvac.ChargeDefectRatio.round(3)
-
-      # Cooling
-      if [HPXML::HVACTypeHeatPumpAirToAir,
-          HPXML::HVACTypeCentralAirConditioner,
-          HPXML::HVACTypeHeatPumpMiniSplit,
-          HPXML::HVACTypeMiniSplitAirConditioner].include?(hvac.CoolType) && hvac.CoolingLoadFraction > 0
-        cool_airflow_rated_defect_ratio = []
-        cool_airflow_rated_ratio = []
-        cool_cfm_m3s = UnitConversions.convert(hvac_sizing_values.Cool_Airflow, 'cfm', 'm^3/s')
+      cool_airflow_rated_defect_ratio = []
+      cool_airflow_rated_ratio = []
+      cool_cfm_m3s = UnitConversions.convert(hvac_sizing_values.Cool_Airflow, 'cfm', 'm^3/s')
+      if not hvac.CoolType == HPXML::HVACTypeHeatPumpGroundToAir
```

Comment:
I see this a few places. How about `!=` instead of `not ... ==`?

Reply:
Sure. 🤦 Sorry for making these happen again..